script.js

```diff
@@ -198,15 +203,24 @@ exit.addEventListener('click', () => {
   contact.classList.remove('blur');
   brand.classList.remove('blur');
 });
+let bool;
 
-function fix() {
-  const mail = document.forms['contact-form'].email.value;
-  const error = document.getElementById('mail-error');
-  if (mail.toLowerCase() !== mail) {
+getinButton.addEventListener('click', () => {
+  bool = false;
+  const mail = emailInput.value;
+  if (mail.toLowerCase() !== mail && bool === false) {
     error.hidden = false;
-    return false;
+  } else if (mail.toLowerCase() === mail) {
+    localStorage.setItem('form', JSON.stringify({
+      name: nameInput.value,
+      email: emailInput.value,
+      textarea: textArea.value,
+    }));
+    bool = true;
```

Comment:
- [ ]  Great job saving the users data to local storage, :+1: :smile: , I also saw your comment on your review request, But according to the project requirement, The user's data are meant to be saved once the user stops typing or once the user leaves the focus of the input filed and not when he/she clicked the button
> When the user changes the content of any input field, the data is saved to the local storage.

Please try to implement this as it is a part of the requirement and also it is good for a great user experience

Reply:
i think is better this way because it will run requirement before saving content, but okay ill try to add that feature @V-Blaze